src/transformers/models/mamba/modeling_mamba.py

```diff
@@ -715,3 +741,156 @@ def forward(
             cache_params=mamba_outputs.cache_params,
             hidden_states=mamba_outputs.hidden_states,
         )
+
+
+class MambaClassificationHead(nn.Module):
```

Comment:
I think this can be copied from somewhere no?

Reply:
The implementation is mostly similar to other ClassificationHead classes of other model types but the ones I checked implemented the class individually. Do you have any abstraction in mind that I can use?

Reply:
I don't think we need to implement the class

Reply:
I'd rather we do like LlamaForSequenceClassification